services-directions/src/main/java/com/mapbox/api/directions/v5/models/BannerInstructions.java

```diff
@@ -156,6 +166,19 @@ public static BannerInstructions fromJson(String json) {
      */
     public abstract Builder sub(@Nullable BannerText sub);
 
+    /**
+     * Additional information that is included
+     * if we feel the driver needs a heads up about something.
+     * Can include information about the next maneuver (the one after the upcoming one),
+     * if the step is short - can be null, or can be lane information.
+     * If we have lane information, that trumps information about the next maneuver.
```

Comment:
Suggested description ⬇️ 
```
Optional image information to display ahead of the upcoming maneuver.
Used to provide a detailed visual for complex junctions.
If the step is short the image should be displayed for the duration of the step, otherwise it is shown as you approach the maneuver.
```

Reply:
Took the suggestion with a few edits, thanks!